crates/bitwarden-crypto/src/sensitive/sensitive.rs

```diff
@@ -54,6 +54,20 @@ impl<V: Zeroize> Sensitive<V> {
     }
 }
 
+/// Helper to convert a `Sensitive<[u8, 16]>` to a `SensitiveVec`.
+impl From<Sensitive<[u8; 16]>> for SensitiveVec {
+    fn from(sensitive: Sensitive<[u8; 16]>) -> Self {
+        SensitiveVec::new(Box::new(sensitive.value.to_vec()))
+    }
+}
+
+/// Helper to convert a `Sensitive<[u8, 32]>` to a `SensitiveVec`.
+impl From<Sensitive<[u8; 32]>> for SensitiveVec {
+    fn from(sensitive: Sensitive<[u8; 32]>) -> Self {
+        SensitiveVec::new(Box::new(sensitive.value.to_vec()))
+    }
+}
```

Comment:
```suggestion
impl<const S: usize> From<Sensitive<[u8; S]>> for SensitiveVec {
    fn from(sensitive: Sensitive<[u8; S]>) -> Self {
        SensitiveVec::new(Box::new(sensitive.value.to_vec()))
    }
}
```
We can replace these two with const generics

Reply:
Nice, I was looking for alternatives but couldn't find any.